README.md

```diff
@@ -1,52 +1,79 @@
+
+## 📌 Overview
+
+This repository contains code, benchmarks, and examples from Xarray on GPUs hackathon project during the
 [NREL/NCAR/NOAA Open Hackathon](https://www.openhackathons.org/s/siteevent/a0CUP00000rwYYZ2A2/se000355)
-in Golden, Colorado from 18-27 February 2025.
+in Golden, Colorado from 18-27 February 2025. The goal of this project is to provide a proof-of-concept example of optimizing the performance of geospatial machine learning workflows on GPUs by using [Zarr-python v3](https://zarr.dev/)  and [NVIDIA DALI](https://developer.nvidia.com/dali). 
 
-# Getting started
+📖 [Read the full blog post](https://xarray.dev/blog/gpu-pipeline)
 
-## Installation
+In this project, we demonstrate how to:
 
-### Basic
+- Optimize chunking strategies for Zarr datasets
+- Read ERA5 Zarr v3 data directly into GPU memory using CuPy and KvikIO
+- Apply GPU-based decompression using NVIDIA's nvCOMP
+- Build end-to-end GPU-native DALI pipelines
+- Improve training throughput for U-Net-based ML models
 
-To help out with development, start by cloning this [repo-url](/../../)
 
-    git clone <repo-url>
+## 📂 Repository Structure
 
-Then I recommend [using mamba](https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html)
-to install the dependencies. A virtual environment will also be created with Python and
-[JupyterLab](https://github.com/jupyterlab/jupyterlab) installed.
+In this repository, you will find the following:
 
-    cd ncar-hackathon-xarray-on-gpus
-    mamba env create --file environment.yml
+- `benchmarks/`: Scripts to evaluate read and write performance for Zarr v3 datasets on both CPU and GPU.
+- `zarr_dali_example/`: Contains a minimal example of using DALI to read Zarr data and train a model. 
+- `zarr_ML_optimization`: Contains an example benchmark for training a U-Net model using DALI with Zarr data format.
+- `rechunk` : Contains a notebook that demonstrates how to optimize chunking strategies for Zarr datasets.
 
-Activate the virtual environment first.
+See [zarr_ML_optimization/README.md](zarr_ML_optimization/README.md) for more details on running the U-Net training example.
 
-    mamba activate gpuhackathon
 
-Finally, double-check that the libraries have been installed.
+# Creating the Environment
 
-    mamba list
+## Basic
 
-### Advanced
+Start by cloning the repo & setting up the `conda` environment:
+```bash
+git clone https://github.com/pangeo-data/ncar-hackathon-xarray-on-gpus.git
+cd ncar-hackathon-xarray-on-gpus
+conda env create --file environment.yml
+conda activate gpuhackathon
+```
+
+### Advanced using `conda-lock`
 
 This is for those who want full reproducibility of the virtual environment.
 Create a virtual environment with just Python and conda-lock installed first.
 
+```
     mamba create --name gpuhackathon python=3.11 conda-lock=2.5.7
     mamba activate gpuhackathon
```

Comment:
Use conda instead of mamba for consistency with above.
```suggestion
conda create --name gpuhackathon python=3.11 conda-lock=2.5.7
conda activate gpuhackathon
```